api/controllers.js

```diff
@@ -24,21 +24,30 @@ module.exports = {
   },
 
   putEditProfile: (req, res) => {
-    // Do your stuff here...
-    res.sendStatus(200);
+    return db.putEditProfile(req.query.data)
+      .then((response) => {
+        if (!response || response.rowCount === 0) {
+          res.status(400).send('Unable to update user profile based on data');
+        } else {
+          res.status(200).send(response.rows);
```

Comment:
Curious to see what this sends. A list of rows affected- or some other relevant info

Reply:
I assume the edit form won't have required fields, so leaving it up to the user. We can do an array or object with all of the edit values. And if e.g. age is not updated, have the current value for age in that response object.

Reply:
👍